content/influxdb/v2.0/reference/flux/stdlib/experimental/last.md

```diff
@@ -31,12 +31,6 @@ experimental.last()
 `experimental.last()` drops empty tables.
 {{% /warn %}}
 
-## Parameters
-
-### tables
-Input data.
-Default is pipe-forwarded data.
-
```

Comment:
These `tables` parameters need to remain. In fact, these will be added to all functions that take piped-forward data as input (see https://github.com/influxdata/docs-v2/issues/2392). There has been some confusion over this. Until recently, we've left this parameter undocumented, but have introduced it back in on newly documented functions.

The following scripts are functionally the same:

```javascript
import "array"
import "experimental"

data = array.from(rows: [{_value: 3.1},{_value:2,8},{_value:4.6}])

data
  |> experimental.last()
```

``` javascript
import "array"
import "experimental"

data = array.from(rows: [{_value: 3.1},{_value:2,8},{_value:4.6}])

experimental.last(tables: data)
```